webapp/src/js/services/db-sync.js

```diff
@@ -33,34 +33,23 @@ angular
     };
 
     var getOptions = function(direction) {
-      var options = {
-        live: true,
-        retry: true,
-        heartbeat: 10000,
-        back_off_function: backOffFunction,
-      };
+      var options = {};
       if (direction === 'to') {
         options.checkpoint = 'source';
         options.filter = readOnlyFilter;
-        return options;
-      } else {
-        // TODO reenable this when single sided checkpointing is fixed:
-        //      https://github.com/pouchdb/pouchdb/issues/6730
-        // options.checkpoint = 'target';
-        return options;
       }
+
+      // TODO reenable this for 'from' replications when single sided checkpointing is fixed:
+      //      https://github.com/pouchdb/pouchdb/issues/6730
+      // options.checkpoint = 'target';
```

Comment:
This used to be in an `else` block so now it would be incorrect if uncommented. I think we've decided to not do this anyway as it could lead to lost updates, so just remove the comment altogether.